templates/jet/.prettierrc.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   singleQuote: true,
   printWidth: 100,
   trailingComma: 'all',
+  arrowParens: 'avoid',
```

Comment:
Same question - could we change this to `always`?

Reply:
Same as above - this was part of the React Native upgrade for some reason. I'm with you though, I'd rather have parentheses always.